src/block/progress-circle/save.js

```diff
@@ -0,0 +1,79 @@
+import ProgressCircleStyles from './style'
+import {
+	BlockDiv,
+	CustomCSS,
+	Typography,
+	getResponsiveClasses,
+	getTypographyClasses,
+	getAlignmentClasses,
+} from '~stackable/block-components'
+import { version as VERSION } from 'stackable'
+import { withVersion } from '~stackable/higher-order'
+import classnames from 'classnames'
+import striptags from 'striptags'
+
+import { compose } from '@wordpress/compose'
+
+const Save = props => {
+	const { className, attributes } = props
+	const responsiveClass = getResponsiveClasses( attributes )
+	const blockAlignmentClass = getAlignmentClasses( attributes )
+	const textClasses = getTypographyClasses( attributes )
+
+	const blockClassNames = classnames( [
+		className,
+		'stk-block-progress-circle',
+		responsiveClass,
+		blockAlignmentClass,
+	] )
+
+	const textClassNames = classnames( [
+		'stk-progress-circle__inner-text',
+		textClasses,
+	] )
+
+	const divClassNames = classnames( [
+		'stk-progress-circle',
+		{
+			'stk--with-animation': attributes.progressAnimate,
+		},
+	] )
+
+	const derivedValue = `${ attributes.textPrefix.trim() }${ attributes.progressPercent }${ attributes.textSuffix.trim() }`.trim()
+
+	return (
+		<BlockDiv.Content
+			className={ blockClassNames }
+			attributes={ attributes }
+		>
+			<ProgressCircleStyles.Content { ...props } />
+			<CustomCSS.Content attributes={ attributes } />
+			<div
+				className={ divClassNames }
+				role="progressbar"
+				aria-valuemin="0"
+				aria-valuemax="100"
+				aria-valuenow={ attributes.progressPercent }
+				aria-valuetext={ striptags( attributes.progressAriaValueText || undefined ) }
+			>
+				<svg>
+					<circle className="stk-progress-circle__background" />
+					<circle className="stk-progress-circle__bar" />
+				</svg>
+				{ attributes.show && (
+					<div className="number">
+						<Typography.Content
+							tagName="h4"
```

Comment:
Let's make this into a span. Headings will be incorrectly picked up by the ToC block and SEO.
```suggestion
							tagName="span"
```